server/src/test/java/org/elasticsearch/action/support/replication/BroadcastReplicationTests.java

```diff
@@ -204,12 +205,15 @@ public void testShardsList() throws InterruptedException, ExecutionException {
         assertThat(shards.get(0), equalTo(shardId));
     }
 
-    private class TestBroadcastReplicationAction extends TransportBroadcastReplicationAction<DummyBroadcastRequest, BroadcastResponse, BasicReplicationRequest, ReplicationResponse> {
-        protected final Set<Tuple<ShardId, ActionListener<ReplicationResponse>>> capturedShardRequests = ConcurrentCollections.newConcurrentSet();
+    private class TestBroadcastReplicationAction extends TransportBroadcastReplicationAction<DummyBroadcastRequest, BroadcastResponse,
+            BasicReplicationRequest, ReplicationResponse> {
+        protected final Set<Tuple<ShardId, ActionListener<ReplicationResponse>>> capturedShardRequests =
+            ConcurrentCollections.newConcurrentSet();
 
         TestBroadcastReplicationAction(Settings settings, ClusterService clusterService, TransportService transportService,
                 ActionFilters actionFilters, IndexNameExpressionResolver indexNameExpressionResolver,
-                TransportReplicationAction<BasicReplicationRequest, BasicReplicationRequest, ReplicationResponse> replicatedBroadcastShardAction) {
+                TransportReplicationAction<BasicReplicationRequest, BasicReplicationRequest, ReplicationResponse>
```

Comment:
Wow! I've never thought of breaking the type from the name before but it looks like we kind of have to to fit it in.

Reply:
yeah, I'm really not happy with doing it this way, but the only alternative I saw was to split the type parameters out to separate lines, which looked worse to me.  I'm open to suggestions.